apps/chat-e2e/src/testData/expectedConstants.ts

```diff
@@ -64,6 +64,7 @@ export const ExpectedConstants = {
     'Model changing will stop sharing and other users will no longer see this conversation.',
   renameSharedPromptMessage:
     'Renaming will stop sharing and other users will no longer see this prompt.',
+  unshareFolderMessage: 'Are you sure that you want to unshare this folder?',
```

Comment:
Shouldn't we have 2 separate constants for conversations and prompts?

Reply:
no, here we are talking about shared folder and messages are identical in this case